src/Components/OpenInExploreLogsButton/OpenInExploreLogsButton.tsx

```diff
@@ -0,0 +1,76 @@
+import { config, useReturnToPrevious } from '@grafana/runtime';
+import { LinkButton } from '@grafana/ui';
+import React, { useMemo } from 'react';
+import { escapeURLDelimiters, replaceEscapeChars, replaceSlash, stringifyAdHocValues } from 'services/extensions/links';
+import { OpenInExploreLogsButtonProps } from './types';
+import { AbstractLabelOperator } from '@grafana/data';
+import { LabelFilterOp } from 'services/filterTypes';
+
+const operatorMap = {
+  [AbstractLabelOperator.Equal]: LabelFilterOp.Equal,
+  [AbstractLabelOperator.NotEqual]: LabelFilterOp.NotEqual,
+  [AbstractLabelOperator.EqualRegEx]: LabelFilterOp.RegexEqual,
+  [AbstractLabelOperator.NotEqualRegEx]: LabelFilterOp.RegexNotEqual,
+};
+
+export default function OpenInExploreLogsButton({
+  datasourceUid,
+  labelMatchers,
```

Comment:
Since we only set the `VAR_LABELS` variable, can we rename this to `streamSelectors`? `VAR_LABELS` will only be used for indexed labels/stream selectors.